e2e/wdio/test/helpers/edit-configs.js

```diff
@@ -0,0 +1,40 @@
+const fs = require('fs');
+
+class EditConfigs {
+  editWebConfig(filepath, serverSideRenderingEndpointUrl) {
+    let webConfig = fs.readFileSync(filepath, 'utf8');
+    let serverSideRenderingEndpoint = webConfig.match(
+      /(?<=serverSideRenderingEngineEndpointUrl=")(.*)(?=")/g
+    )[0];
+    webConfig = webConfig.replace(serverSideRenderingEndpoint, serverSideRenderingEndpointUrl);
+    fs.writeFileSync(filepath, webConfig);
+  }
+
+  editDotEnv({ filepath, jssEditingSecret, apiKey, graphQLEndpoint, rootItemPath } = {}) {
```

Comment:
For the case of environment variables, editing the .env file is something that would only be done on a dev machine. If the app is executing in other environments, then the variables would be provided in other ways. This is not JSS-specific functionality. So, I think it's ok to just pass in the necessary env variables when we call jss commands, rather than trying to automate this.